zio-http/shared/src/main/scala/zio/http/codec/HttpContentCodec.scala

```diff
@@ -181,6 +180,18 @@ object HttpContentCodec {
     }
   }
 
+  object html {
+    implicit val htmlCodec: HttpContentCodec[Dom] = {
```

Comment:
Implicits should not have to be imported. This means this should be located in the companion object of `Dom` or in the companion object of `HttpContentCodec`.